packages/fast-components-styles-msft/src/listbox-item/index.ts

```diff
@@ -0,0 +1,113 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { ListboxItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { density } from "../utilities/density";
+import { defaultHeight, maxHeight, minHeight } from "../utilities/height";
+import {
+    disabledContrast,
+    ensureForegroundNormal,
+    ensureNormalContrast,
+    hoverContrast,
+} from "../utilities/colors";
+import { applyFocusVisible } from "@microsoft/fast-jss-utilities";
+import { applyTypeRampConfig } from "../utilities/typography";
+import typographyPattern from "../patterns/typography";
+import { contrast, toPx } from "@microsoft/fast-jss-utilities";
+import { curry } from "lodash-es";
+
+const styles: ComponentStyles<ListboxItemClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<ListboxItemClassNameContract, DesignSystem> => {
+    type ContrastFunction = (operandColor: string, referenceColor: string) => string;
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const contrastScale: number = designSystem.contrast;
+    const brandColor: string = designSystem.brandColor;
+    const color: string = designSystem.foregroundColor;
+    const scaledEnsureNormalContrast: ContrastFunction = curry(ensureNormalContrast)(
+        contrastScale
+    );
+    const primaryRestBackgroundColor: string = scaledEnsureNormalContrast(
+        scaledEnsureNormalContrast(brandColor, designSystem.backgroundColor),
+        color
+    );
+    const primaryDisabledBackground: string = disabledContrast(
+        contrastScale,
+        primaryRestBackgroundColor,
+        designSystem.backgroundColor
+    );
+    const primaryDisabledColor: string = disabledContrast(
+        contrastScale,
+        color,
+        primaryDisabledBackground
+    );
+    const primarySelectedBackground: string = contrast(
+        1.7,
+        designSystem.foregroundColor,
+        designSystem.backgroundColor
+    );
+    return {
+        listboxItem: {
+            listStyleType: "none",
+            height: density(32),
+            display: "grid",
```

Comment:
@chrisdholt do you have an opinion on display grid vs flexbox?

Reply:
We use grid in context menu item for this also.  (Personally I find grids much, much easier to control than flexbox)

Reply:
That may be true, but flexbox is more performant

Reply:
That doesn't seem to always be the case (and I'm not sure the difference is significant) - https://blogs.igalia.com/jfernandez/2015/06/24/performance-on-grid-layout/